utils/utils.go

```diff
@@ -60,6 +56,9 @@ func Init() {
 		color.Magenta("Using local recipe directory: %s", color.HiBlackString(recipeDir))
 		WebmanRecipeDir = recipeDir
 	}
+	if err := os.MkdirAll(WebmanRecipeDir, os.ModePerm); err != nil {
```

Comment:
The tests already helped catch this bug, where the recipes dir wasn't made ahead of time. 
This means currently new installs of webman are broken.

Reply:
Great catch, and nice job getting these started. There's plenty of areas for testing that probably will catch more.